src/Apps/Home/Components/HomeContentCard.tsx

```diff
@@ -0,0 +1,186 @@
+import { CaptionedImage as BrazeContentCard } from "@braze/web-sdk"
+import * as React from "react"
+import {
+  Box,
+  Image,
+  ResponsiveBox,
+  Spacer,
+  Text,
+  Button,
+  GridColumns,
+  Column,
+} from "@artsy/palette"
+import { cropped } from "Utils/resized"
+import { RouterLink } from "System/Router/RouterLink"
+import { Media } from "Utils/Responsive"
+import { HomeHeroUnitCredit } from "./HomeHeroUnits/HomeHeroUnitCredit"
+
+interface HomeContentCardProps {
+  card: BrazeContentCard
+  index: number
+}
+
+export const HomeContentCard: React.FC<HomeContentCardProps> = ({
+  card,
+  index,
+}) => {
+  const extras = card.extras || {}
+
+  const image = cropped(card.imageUrl!, {
+    width: 1270,
+    height: 500,
+  })
+
+  const cardLink = card.url ?? ""
+
+  const handleClick = () => {
+    const appboy = (window as any).appboy
```

Comment:
Can we update the Window type to include appboy?

Reply:
And should we alias the variable name to `const braze` so that folks don't get confused in the future?

Reply:
What if I promise to return to this once we've finished upgrading our segment integration?? We're on an older version of the web sdk and the next major release from them switches to `window.braze` so then I could do all that stuff and it'll finally make sense. 😝 

I'm hoping to get that migration done any week now if I can just get 3 vendors to play nice (OneTrust, Segment, Braze). 😅

Reply:
Oh yeah np!